### PR TITLE
Fixing README.md

### PR DESCRIPTION
**Pull request by picrin**
# Background
I've improved README.md as part of getting the platform to run locally

# Changes
No changes of the code, just README.md

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
I've made no changes to the code
### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

### Approvers
- [ ] Approved by a member of the core engineering team

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ Make sure that you clone this repo and are in the `ui` folder. then simply do:
 
 See more available commands like syntax checking with `make help`.
 
+Once UI is running, open your browser and go to `localhost:5000`.
```
Comment (robioson) [inline README.md]: You could make this a link: http://localhost:5000

**Comment by marcellp** [comment]
Yup, @robioson is right, other than that lgtm. let me know when you've made the changes so I can approve.

**Review by kafkasl — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -20,6 +20,8 @@ Make sure that you clone this repo and are in the `ui` folder. then simply do:
 
 See more available commands like syntax checking with `make help`.
 
+Once UI is running, open your browser and go to [localhost:5000](localhost:5000).
+
```
Comment (robioson) [inline README.md]: Duplicated below
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -29,6 +31,8 @@ To get it running end-to-end locally with mocked dataset, you will need to set u
 
 Just follow the README of each of them for instructions on how to get it to run.
 
+Once you have everything running on the backend, go to `localhost:5000`. You will get redirected to an authentication page. You will have to create a staging account by clicking through "Sign up" link (even if you have an existing production account).
```
Comment (robioson) [inline README.md]:
```suggestion
Once you have everything running on the backend, go to <http://localhost:5000> in your browser. You will get redirected to an authentication page. You will have to create a staging account by clicking through "Sign up" link (even if you have an existing production account).
```

File: README.md [hunk 4]
```diff
@@ -75,4 +79,4 @@ You can check for the size of the bundles served to the user and loaded onto the
 Alternatively, you can use `npm run analyze:server` or `npm run analyze:client` to get information about
 bundle sizes for just server-side rendered and client-side scripts.
 
-The script will produce HTML artifacts that will open in your default browswer as a Voronoi treemap.
\ No newline at end of file
+The script will produce HTML artifacts that will open in your default browswer as a Voronoi treemap.
```
Comment (robioson) [inline README.md]:
```suggestion
The script will produce HTML artifacts that will open in your default browser as a Voronoi treemap.
```

File: README.md [hunk 5]
```diff
@@ -29,6 +29,8 @@ To get it running end-to-end locally with mocked dataset, you will need to set u
 
 Just follow the README of each of them for instructions on how to get it to run.
 
+Once you have everything running on the backend, go to <http://localhost:5000> in your browser. You will get redirected to an authentication page. You will have to create a staging account by clicking through "Sign up" link (even if you have an existing production account).
```
Comment (robioson) [inline README.md]:
```suggestion
Once you have everything running on the backend, go to <http://localhost:5000> in your browser.
You will get redirected to an authentication page. You will have to create a staging account by clicking
through "Sign up" link (even if you have an existing production account).
```

**Pull request merged**